docs/getting-started/example-deployment/running-the-deployment.md

```diff
@@ -0,0 +1,235 @@
+# Running the Deployment
```

Comment:
It seems this follows the `project-setup.md` page directly. Could these be the same page? Also it would be good to have a landing page to explain the example deployment section and the order in which the pages should be navigated.

Reply:
That's not strictly true I've outlined the new site structure at top of this PR, github will just be showing the files in alphabetical order. If you run the site up you should be able to see the new navigation path.

Reply:
Makes sense could probably add the additional config page into one of the others